{"ast":null,"code":"\"TURBOPACK { transition: next-shared }\";\n\"use strict\";\n\nObject.defineProperty(exports, \"__esModule\", {\n  value: true\n});\n0 && (module.exports = {\n  getExpectedRequestStore: null,\n  requestAsyncStorage: null\n});\nfunction _export(target, all) {\n  for (var name in all) Object.defineProperty(target, name, {\n    enumerable: true,\n    get: all[name]\n  });\n}\n_export(exports, {\n  getExpectedRequestStore: function () {\n    return getExpectedRequestStore;\n  },\n  requestAsyncStorage: function () {\n    return _requestasyncstorageinstance.requestAsyncStorage;\n  }\n});\nconst _requestasyncstorageinstance = require(\"./request-async-storage-instance\");\nfunction getExpectedRequestStore(callingExpression) {\n  const store = _requestasyncstorageinstance.requestAsyncStorage.getStore();\n  if (store) return store;\n  throw new Error(\"`\" + callingExpression + \"` was called outside a request scope. Read more: https://nextjs.org/docs/messages/next-dynamic-api-wrong-context\");\n}\nif ((typeof exports.default === 'function' || typeof exports.default === 'object' && exports.default !== null) && typeof exports.default.__esModule === 'undefined') {\n  Object.defineProperty(exports.default, '__esModule', {\n    value: true\n  });\n  Object.assign(exports.default, exports);\n  module.exports = exports.default;\n}","map":{"version":3,"names":["getExpectedRequestStore","requestAsyncStorage","_requestasyncstorageinstance","callingExpression","store","getStore","Error"],"sources":["../../../src/client/components/request-async-storage.external.ts"],"sourcesContent":[null],"mappings":"AAQE;;;;;;;;;;;;;;;;;EAgBcA,uBAAuB,WAAAA,CAAA;WAAvBA,uBAAA;;EAFPC,mBAAmB,WAAAA,CAAA;WAAnBC,4BAAA,CAAAD,mBAAmB;;;6CAbQ;AAe7B,SAASD,wBAAwBG,iBAAyB;EAC/D,MAAMC,KAAA,GAAQF,4BAAA,CAAAD,mBAAmB,CAACI,QAAQ;EAC1C,IAAID,KAAA,EAAO,OAAOA,KAAA;EAClB,MAAM,IAAIE,KAAA,CACR,GAAC,GAAIH,iBAAA,GAAkB;AAE3B","ignoreList":[]},"metadata":{},"sourceType":"script","externalDependencies":[]}